Source/Actions/ActionCell.swift

```diff
@@ -5,6 +5,8 @@ final class ActionCell: UICollectionViewCell {
     @IBOutlet private(set) var titleLabel: UILabel!
     @IBOutlet private var highlightedBackgroundView: UIView!
 
+    private var titleLabelColorFromVisualStyle:UIColor?
```

Comment:
How about we call this `textColor` for simplicity?

And a nitpick: could you add a space after the `:`?